lib/getLatestRelease.js

```diff
@@ -73,9 +73,11 @@ const prepAssets = assets => {
   const preppedAssets = assets
     .filter(item => {
       // Omit OS X ZIP files (used for update purposes)
+      // Omit electron packages
       return !(
-        item.name.indexOf('.zip') !== -1 && item.name.indexOf('darwin') !== -1
-      );
+        (includes(item.name, '.zip') &&
+        includes(item.name, 'darwin')) ||
+        includes(item.name, 'electron'))
```

Comment:
Rather than filtering out `electron`, I think it might be better to filter out files ending in `.deb` or `.rpm`.